components/automate-ui/src/app/pages/api-token/details/api-token-details.component.ts

```diff
@@ -45,9 +46,10 @@ export class ApiTokenDetailsComponent implements OnInit, OnDestroy {
         filter(identity),
         takeUntil(this.isDestroyed))
         .subscribe((state) => {
-          this.token = <ApiToken>Object.assign({}, state);
-          this.status = this.token.active ? 'Active' : 'Inactive';
-          this.updateNameForm.controls['name'].setValue(this.token.name);
+          this.token = { ...state };
```

Comment:
Great find! Thank you for this gem. (By the way, in JavaScript it is _spread_. Ruby, PowerShell (others?) call it _splat_.) 😉